Cargo.toml

```
@@ -18,12 +18,17 @@ chrono = { version = "0.4", default-features = false, features = [
     "std",
     "serde",
 ] }
+eyre = "0.6.12"
+reqwest = { version = "0.12.1" , features = ["json"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+sysinfo = "0.30.7"
+tokio = { version = "1.36.0", features = ["full"] }
```

```suggestion
tokio = "1.36"
```

Please only pick the features you'll need. (see my next comment)